### PR TITLE
Updated Angular to v4.4.4

### PR DESCRIPTION
**Pull request by excelulous**
Also changed tsconfig options to fix an error that arose - TS2304: Cannot find name 'Iterable'

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1161 @@
+{
+  "name": "ngx-embed-video",
```
Comment (SamirHodzic) [inline package-lock.json]: First comment, please remove `package-lock.json`.

**Comment by excelulous** [comment]
It's auto-generated by npm and is intended to be committed - https://docs.npmjs.com/files/package-lock.json

Do you still want me to remove it?

**Comment by SamirHodzic** [comment]
I know and you are right, but I found that it adds a lot of noise to diff so don't wanna to introduce it to this tiny plugin.

**Comment by excelulous** [comment]
Ok, no problem.

**Comment by excelulous** [comment]
I removed package-lock.json

**Comment by SamirHodzic** [comment]
Seems like your PR have conflicts now.

**Pull request merged**